backend/server3.js

```diff
@@ -0,0 +1,206 @@
+var express = require('express');
```

Comment:
Codacy has a fix for the issue: [Replace `'express'` with `"express"`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
var express = require("express");
```